tests/testthat/test.ifs.R

```diff
@@ -1,98 +1,95 @@
-library(testthat)
-library(hablar)
-
-context("ifs")
-test_that("if_else_", {
-  # No missing argument
-  expect_equal(if_else_(c(T, F, NA),
-                        as.numeric(c(1, 2, 3)),
-                        as.numeric(c(99))),
-               as.numeric(c(1, 99, NA))
-  )
-  expect_equal(if_else_(c(T, F, NA),
-                        as.numeric(c(1, 2, 3)),
-                        as.numeric(c(4, 5, 6))),
-               as.numeric(c(1, 5, NA))
-  )
-  expect_equal(if_else_(c(T, F, NA),
-                        as.numeric(c(1, 2, 3)),
-                        NA),
-               as.numeric(c(1, NA, NA))
-  )
-  expect_equal(if_else_(c(T, F, NA),
-                        NA,
-                        "hey"),
-               as.character(c(NA, "hey", NA))
-  )
-  expect_equal(if_else_(c(T, F, NA),
-                        NA,
-                        c(1L, 2L, 3L)),
-               as.integer(c(NA, 2L, NA))
-  )
-  expect_equal(if_else_(c(T, F, NA),
-                        NA,
-                        as.factor(c("A", "B", "C"))),
-               factor(c(NA, "B", NA), levels = c("A", "B", "C"))
-               )
-  expect_equal(if_else_(c(T, F, NA),
-                        as.factor(c("A", "B", "C")),
-                        NA),
-               factor(c("A", NA, NA), levels = c("A", "B", "C"))
-  )
-  expect_equal(if_else_(c(T, F, NA),
-                        as.Date(c("2019-01-01", "2019-01-01", "2019-01-01")),
-                        as.Date("2018-01-01")),
-               as.Date(c("2019-01-01", "2018-01-01", NA))
-  )
-  expect_equal(if_else_(c(T, F, NA),
-                        as.Date(c("2019-01-01", "2019-01-01", "2019-01-01")),
-                        NA),
-               as.Date(c("2019-01-01", NA, NA)) 
-  )
-
-  expect_error(if_else_(c(T, F, NA),
-                        1,
-                        1L))

```

Comment:
I'm not sure why the diff is so large. Maybe something about spaces. This is all i removed